### PR TITLE
[Feature] Enable ppl visualization in Chatbot

### PR DESCRIPTION
**Pull request by SuZhou-Joe**
### Description

Enable ppl_visualization with Chatbot.

image

image

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by SuZhou-Joe** [comment]
@joshuali925 Would you mind taking a look on this PR?

File: public/types.ts [hunk 1]
```diff
@@ -34,3 +33,11 @@ export interface ObservabilitySetup {}
...
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface ObservabilityStart {}
+
+/**
+ * Introduce a compile dependency on dashboards-assistant
+ * as observerability need some types from the plugin.
+ * It will gives an type error when dashboards-assistant is not installed so add a ts-ignore to suppress the error.
+ */
+// @ts-ignore
+export type { AssistantSetup } from "../../dashboards-assistant/public";
```
Comment (joshuali925) [inline public/types.ts:43]: also add this to https://github.com/opensearch-project/dashboards-observability/blob/1f9cc08be2398b1fe8bc6c20075674956857be05/public/types.ts#L23-L29?
`assistantDashboards?: AssistantSetup;`

**Review by joshuali925 — APPROVED** [review]
minor comment, thanks

File: public/types.ts (hunk 1 above)
Comment (SuZhou-Joe) [reply public/types.ts:45]: Sure, done for this.

**Comment by SuZhou-Joe** [comment]
One more approval is needed, can anyone help to review please?

**Comment by joshuali925** [comment]
maybe @paulstn? could you help to review

**Review by mengweieric — APPROVED** [review]

**Comment by SuZhou-Joe** [comment]
@joshuali925 , could you please add `backport 2.x` label?

**Pull request merged**